src/partials/Products.html

```diff
@@ -0,0 +1,78 @@
+
+<!DOCTYPE html>
```

Comment:
Te rog sa faci modificarile necesare pentru a respecta modelul de upload de acum, de exemplu in document trebuie sa ai doar tagurile de section si continutul lor, tot ce este extra trebuie sters.. adica doctype html head body etc